src/id3manager/metadata.py

```diff
@@ -0,0 +1,97 @@
+import typing as t
+
+import mutagen.id3 as id3
+
+from . import utils
+from .formats import get_reader, get_writer
+
+
+__all__ = [
+    "parse_metadata",
+    "render_metadata",
+    "ID3SourceFrameOrder",
+]
+
+
+def parse_metadata(fileobj: t.IO, format: str, audio_len: float) -> t.List[id3.Frame]:
```

Comment:
nit: It'd be nice to have consistent naming. We should rather rename parse_metadata/render_metadata into read_metadata/write_metadata or rename MetadataReader/MetadataWriter into parser/renderer counterparts.

Reply:
Done!